modules/webrequest-pipeline/sources/webrequest-context.es

```diff
@@ -128,6 +136,6 @@ export default class WebRequestContext {
   }
 
   isBackgroundRequest() {
-    return this.tabId === -1;
+    return this.tabId === -1 && this.originUrlParts.protocol.endsWith('extension:');
```

Comment:
How to detect extension request is rather a detail, but I wonder if this approach is actually better than checking for Chromium like in the previous version:

https://github.com/ghostery/common/pull/82/files#diff-6ec83f4145502c9abc870433bd9ca41a93a0b71bbafb35fd59c8766011416975R131

On Firefox, tabId will be -1. And once we start matching again these strings, you have to make assumptions about the browser anyway. But you will never reach this check with Firefox, as I understand.